tasks/engine_setup.yml

```diff
@@ -0,0 +1,61 @@
+---
+- block:
+  - name: Use the default answerfile
+    template:
+      src: answerfile_{{ ovirt_engine_setup_version }}_basic.txt.j2
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is undefined
+
+  - name: Copy custom answer file
+    no_log: "True"
+    template:
+      src: "{{ ovirt_engine_setup_answer_file_path }}"
+      dest: /tmp/answerfile.txt
+      mode: 0600
+      owner: root
+      group: root
+    when: ovirt_engine_setup_answer_file_path is defined
+
+  - name: Update setup packages
+    shell: 'yum -y update ovirt*setup*'
+    when: ovirt_engine_setup_update_setup_packages
+    tags:
+      - skip_ansible_lint
+
+  - name: Update all packages
+    shell: 'yum -y update'
+    when: ovirt_engine_setup_update_all_packages
+    tags:
+      - skip_ansible_lint
```

Comment:
What this tag means?

Reply:
https://github.com/willthames/ansible-lint#false-positives

It's from ansible-lint checking.

Because I think linters will compain I am not using yum module :) Will confirm, I it will be needed to keep this instead of yum module.

Reply:
Well, if possible we shouldn't use it, and use correct format. And if we need to use it we should document why.

Reply:
I already removed and I am using yum module. I think you see outdated page.